src/com/intellij/idea/plugin/hybris/common/services/impl/DefaultCommonIdeaService.kt

```diff
@@ -179,62 +142,29 @@ class DefaultCommonIdeaService : CommonIdeaService {
         connectionSettings.hostIP = connectionSettings.hostIP?.replace(regex, "")
     }
 
-    /*private fun getLocalProperties(project: Project): Properties? {
-        val configDir = HybrisProjectSettingsComponent.getInstance(project).state.configDirectory ?: return null
-        val propFile = File(configDir, HybrisConstants.LOCAL_PROPERTIES)
-        if (!propFile.exists()) {
-            return null
-        }
-        val prop = Properties()
-        try {
-            FileReader(propFile).use { fr ->
-                prop.load(fr)
-                return prop
-            }
-        } catch (e: IOException) {
-            LOG.info(e.message, e)
-        }
-        return null
-    }*/
-
-    private fun matchAllModuleNames(
-        namePatterns: Collection<String>,
-        moduleNames: Collection<String>
-    ): Boolean {
-        return namePatterns.stream()
-            .allMatch { pattern: String -> matchModuleName(pattern, moduleNames) }
-    }
-
     private fun matchModuleName(pattern: String, moduleNames: Collection<String>): Boolean {
         val regex = Regex("\\Q$pattern\\E".replace("*", "\\E.*\\Q"))
         return moduleNames.stream()
             .parallel()
-            .anyMatch { p: String -> p.matches(regex) }
+            .anyMatch { it.matches(regex) }
     }
 
     private fun getUrl(settings: HybrisRemoteConnectionSettings?): String {
         val ip = settings!!.hostIP
-        val sb = StringBuilder()
-        if (settings.isSsl) {
-            sb.append(HybrisConstants.HTTPS_PROTOCOL)
-        } else {
-            sb.append(HybrisConstants.HTTP_PROTOCOL)
-        }
-        sb.append(ip)
-        sb.append(HybrisConstants.URL_PORT_DELIMITER)
-        sb.append(settings.port)
-        val hac = settings.hacWebroot
-        if (StringUtils.isNoneBlank(hac)) {
-            sb.append('/')
-            sb.append(StringUtils.strip(hac, " /"))
-        }
-        val result = sb.toString()
-        LOG.debug("Calculated hostHacURL=$result")
-        return result
-    }
+        val urlBuilder = StringBuilder()
```

Comment:
here, StringBuilder can be used, but same can be also achieved with Kotlin StringTemplates, slightly different approach